src/main/java/com/karankumar/bookproject/ui/book/BookForm.java

```diff
@@ -168,6 +172,9 @@ private void configureBinder() {
         binder.forField(shelf)
                 .withValidator(Objects::nonNull, "Please select a shelf")
                 .bind("shelf.shelfName");
+        binder.forField(bookSeries).withValidator(series -> (series == null || series > 0),
+                "Book series should not be negative")
```

Comment:
Since 0 is not negative, perhaps a better message would be 'Series position must be at least 1'